src/System.Private.CoreLib/shared/System/Diagnostics/Tracing/EventSource.cs

```diff
@@ -1195,7 +1199,7 @@ protected unsafe void WriteEventWithRelatedActivityIdCore(int eventId, Guid* rel
                     }
 #endif // FEATURE_MANAGED_ETW
 
-                    if (m_Dispatchers != null && m_eventData[eventId].EnabledForAnyListener)
+                            if (m_Dispatchers != null && m_eventData[eventId].EnabledForAnyListener)
```

Comment:
nit, this indentation is off (tabs vs spaces?)

Reply:
Fixed.